### PR TITLE
Fix managed authentication docs

### PR DESCRIPTION
**Pull request by ThumulaPerera**
## Purpose
$subject

### CODE REVIEW

**Review by shaniR — APPROVED** [review]

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 1]
```diff
@@ -223,10 +226,13 @@ You can manage OAuth keys either with the Choreo built-in identity provider, Asg
...
-     5. Click **Regenerate Secret**. 
+     5. Click **Generate Secret**. 
+
+        !!! Note
+             If the **Regenerate Secret** button is shown instead of the **Generate Secret** button, it indicates that OAuth keys are already generated for the component for the selected environment.
 
-        !!! tip
-             If the **Regenerate Secret** button is disabled, it indicates that OAuth keys are already generated for the component for the selected environment.
+    !!! tip
+        If you need to invoke APIs secured with role-based access control, you must create roles in the application and map those roles to relevant permissions (scope). Then those roles should be assigned to user groups. For more information, see [Create roles and assign permissions](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-2-create-roles-and-assign-permissions) and [Assign roles to user groups](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-3-assign-roles-to-user-groups) sections in [Test Secure API Access with Choreo Built-In Security Token Service](../test-secure-api-access-with-choreo-built-in-security-token-service).
```
Comment (omindu) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:235]:
```suggestion
        If you need to invoke APIs secured with role-based access control, you can test this within Choreo itself by creating roles for the application and mapping those roles to relevant permissions (scope) and user groups. For more information, see [Create roles and assign permissions](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-2-create-roles-and-assign-permissions) and [Assign roles to user groups](../test-secure-api-access-with-choreo-built-in-security-token-service/#step-3-assign-roles-to-user-groups) sections in [Test Secure API Access with Choreo Built-In Security Token Service](../test-secure-api-access-with-choreo-built-in-security-token-service).
```

Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md]: Resolved with [b498cef](https://github.com/wso2/docs-choreo-dev/pull/1250/commits/b498cef69958e084f6abf79960aa5f3f33db550f)

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md [hunk 2]
```diff
@@ -274,7 +280,6 @@ You can manage OAuth keys either with the Choreo built-in identity provider, Asg
          2. Add the following as authorized redirect URL.
          3. Specify the following as authorized redirect URLs:
          4. Specify the access token type as JWT.
-         5. Set the refresh token expiry time to 1 day. 
 
             !!! tip
```
Comment (shaniR) [inline en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:284]: this tip is duplicated isn't it?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md (hunk 2 above)
Comment (ThumulaPerera) [reply en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md:284]: This is in a tab view. Similar tips with slight variations (Indicating how to perform the role to group mapping for each type of IdP) is displayed in each tab

**Review by shaniR — APPROVED** [review]

**Pull request merged**